### PR TITLE
SEOImages: User Story 1463880, Part 21

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Files #1463880

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019031700036/Report/
There are five image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

File: docs/core/tutorials/debugging-with-visual-studio.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ Run your program in Debug mode and try a few debugging features:
 
 1. Program execution stops when it reaches the breakpoint and before the `Console.WriteLine` method executes. The **Autos** window displays the values of variables that are used around the current line. The **Locals** window (which you can view by clicking the **Locals** tab) displays the values of variables that are defined in the currently executing method.
 
-   ![Visual Studio application window](./media/debugging-with-visual-studio/break.png)
+   ![Screenshot that shows an example of a breakpoint.](./media/debugging-with-visual-studio/breakpoint-console-window.png)
```
Comment (mairaw) [inline docs/core/tutorials/debugging-with-visual-studio.md]:
```suggestion
   ![Screenshot of a breakpoint in Visual Studio.](./media/debugging-with-visual-studio/breakpoint-console-window.png)
```

File: docs/framework/security/wif-claims-programming-model.md [hunk 2]
```diff
@@ -7,7 +7,7 @@ author: "BrucePerlerMS"
 # WIF Claims Programming Model
 ASP.NET and Windows Communication Foundation (WCF) developers ordinarily use the IIdentity and IPrincipal interfaces to work with the user’s identity information. In .NET 4.5, Windows Identity Foundation (WIF) has been integrated such that claims are now always present for any principal as illustrated in the following diagram:
 
- ![WIF Claims Programming Model](../../../docs/framework/security/media/wifclaimsprogrammingmodel.png "WIFClaimsProgrammingModel")
+ ![Diagram that shows the WIF Claims Programming Model.](./media/wif-claims-programming-model/wif-claims-programming-model.png)
```
Comment (mairaw) [inline docs/framework/security/wif-claims-programming-model.md]:
```suggestion
 ![Diagram that shows the WIF Claims programming model.](./media/wif-claims-programming-model/wif-claims-programming-model.png)
```

File: docs/framework/wcf/feature-details/message-security-with-a-certificate-client.md [hunk 3]
```diff
@@ -11,7 +11,7 @@ The following scenario shows a Windows Communication Foundation (WCF) client and
   
  For a sample application, see [Message Security Certificate](../../../../docs/framework/wcf/samples/message-security-certificate.md).  
   
- ![Client with certificate](../../../../docs/framework/wcf/feature-details/media/clientwithcertificate.gif "ClientWithCertificate")  
+ ![Screenshot that shows a client with certificate.](./media/message-security-with-a-certificate-client/client-with-certificate.gif)
```
Comment (mairaw) [inline docs/framework/wcf/feature-details/message-security-with-a-certificate-client.md]: Can you move this before the sentence in line 12?

File: docs/framework/windows-workflow-foundation/how-to-create-a-sequential-workflow.md [hunk 4]
```diff
@@ -141,9 +141,9 @@ Workflows can be constructed from built-in activities as well as from custom act
     "Your guess is too high."  
     ```  
   
-     The following example illustrates the completed workflow.  
+     The following example illustrates the completed workflow:  
   
-     ![Completed Sequential Workflow](./media/wfsequentialgettingstartedtutorialcomplete.JPG "WFSequentialGettingStartedTutorialComplete")  
+     ![Screenshot that shows a completed sequential workflow.](./media/how-to-create-a-sequential-workflow/complete-sequential-workflow.jpg)
```
Comment (mairaw) [inline docs/framework/windows-workflow-foundation/how-to-create-a-sequential-workflow.md]:
```suggestion
     ![Screenshot that shows the completed sequential workflow.](./media/how-to-create-a-sequential-workflow/complete-sequential-workflow.jpg)  
```

File: docs/framework/winforms/advanced/how-to-draw-opaque-and-semitransparent-lines.md [hunk 5]
```diff
@@ -19,9 +19,9 @@ When you draw a line, you must pass a <xref:System.Drawing.Pen> object to the <x
 ## Example  
  The following example draws a bitmap and then draws three lines that use the bitmap as a background. The first line uses an alpha component of 255, so it is opaque. The second and third lines use an alpha component of 128, so they are semitransparent; you can see the background image through the lines. The statement that sets the <xref:System.Drawing.Graphics.CompositingQuality%2A> property causes the blending for the third line to be done in conjunction with gamma correction.  
   
- The following illustration shows the output of the following code.  
+ The following illustration shows the output of the following code:
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-draw-opaque-and-semitransparent-lines.md]: can you move this to after the sample code?

File: docs/framework/winforms/advanced/how-to-draw-with-opaque-and-semitransparent-brushes.md [hunk 6]
```diff
@@ -18,10 +18,10 @@ When you fill a shape, you must pass a <xref:System.Drawing.Brush> object to one
   
 ## Example  
  The following example draws a bitmap and then fills three ellipses that overlap the bitmap. The first ellipse uses an alpha component of 255, so it is opaque. The second and third ellipses use an alpha component of 128, so they are semitransparent; you can see the background image through the ellipses. The call that sets the <xref:System.Drawing.Graphics.CompositingQuality%2A> property causes the blending for the third ellipse to be done in conjunction with gamma correction.  
+
+ The following illustration shows the output of the following code:
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-draw-with-opaque-and-semitransparent-brushes.md]: same here. can you move this to after the code?

File: docs/framework/winforms/advanced/how-to-draw-with-opaque-and-semitransparent-brushes.md [hunk 7]
```diff
@@ -18,10 +18,10 @@ When you fill a shape, you must pass a <xref:System.Drawing.Brush> object to one
   
 ## Example  
  The following example draws a bitmap and then fills three ellipses that overlap the bitmap. The first ellipse uses an alpha component of 255, so it is opaque. The second and third ellipses use an alpha component of 128, so they are semitransparent; you can see the background image through the ellipses. The call that sets the <xref:System.Drawing.Graphics.CompositingQuality%2A> property causes the blending for the third ellipse to be done in conjunction with gamma correction.  
+
+ The following illustration shows the output of the following code: 
   
- The following illustration shows the output of the following code.  
-  
- ![Opaque and Semitransparent](./media/compqualellipse.png "compqualellipse")  
+ ![Illustration that shows opaque and semitransparent output.](./media/how-to-draw-with-opaque-and-semitransparent-brushes/comp-qual-ellipse-semitransparent.png)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-draw-with-opaque-and-semitransparent-brushes.md]: maybe use the full name of the property?

```suggestion
 ![Illustration that shows opaque and semitransparent output.](./media/how-to-draw-with-opaque-and-semitransparent-brushes/compositingquality-ellipse-semitransparent.png)  
```

File: docs/framework/winforms/advanced/media/how-to-draw-with-opaque-and-semitransparent-brushes/desktop.ini [hunk 8]
```diff
@@ -0,0 +1,2 @@
+[LocalizedFileNames]
```
Comment (mairaw) [inline docs/framework/winforms/advanced/media/how-to-draw-with-opaque-and-semitransparent-brushes/desktop.ini]: we shouldn't be adding this file, right?

File: docs/standard/data/xml/xslt-transformations-with-the-xsltransform-class.md [hunk 9]
```diff
@@ -18,8 +18,7 @@ The goal of the XSLT is to transform the content of a source XML document into a
 
 ## Overview
 
-![XSLT Transformation Architecture](media/xslttransformationswithxsltransformclass.gif "xsltTransformationsWithXslTransformClass")  
-Transformation Architecture
+![Diagram that shows the XSLT Transformation Architecture.](./media/xslt-transformations-with-the-xsltransform-class/xslt-transformation-architecture.gif)
```
Comment (mairaw) [inline docs/standard/data/xml/xslt-transformations-with-the-xsltransform-class.md]:
```suggestion
![Diagram that shows the XSLT transformation architecture.](./media/xslt-transformations-with-the-xsltransform-class/xslt-transformation-architecture.gif) 
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Thanks @damabe. A few comments before we merge,

**Comment by mairaw** [comment]
@damabe can you address my suggestions here?

**Comment by damabe** [comment]
@mairaw I've addressed your requested changes. Please have a look. Updated CATS: https://contentqacats.azurewebsites.net/CheckResults/2019032300033/Report/

There are five image names identified by CATS as being too short. They should be handled at a later time as they are not part of the SEO image without hyphen work.

**Review by mairaw — APPROVED** [review]
Thanks @damabe!

**Pull request merged**